tests/s3_bucket/test_s3_pages_content.py

```diff
@@ -75,21 +77,31 @@ def test_page_content(bucket_books_tree, s3_books_title_version_dict, code_tag,
     for elem in range(len(bucket_books_tree)):
 
         book_id.clear()
-        book_id.append(bucket_books_tree[elem]["id"])
+        uuids_vers = bucket_books_tree[elem]["id"]
+        book_id.append(uuids_vers[0 : uuids_vers.index("@")])
 
         book_title.clear()
         book_title.append(bucket_books_tree[elem]["title"])
 
         title_results = list(s3_books_title_version_dict.keys())
 
+        try:
+
+            # compares book uuids in the queue-state list against actual books in aws s3 bucket
+            assert [j for j in book_id if j in s3_books_uuids]
+
+        except AssertionError as as_error:
+
+            print(f"{as_error}: book id {book_id} : {book_title} not found, next item")
+
         try:
 
             # compares book titles in the queue-state list against actual books in aws s3 bucket
             assert [i for i in book_title if i in title_results]
 
         except AssertionError as as_error:
 
-            print(f"{as_error}: book title {book_title} not found, next item")
+            print(f"{as_error}: book title {book_title} : {book_id} not found, next item")
```

Comment:
To your question, I do think this is probably a meaningless check. If it fails, it means we won't check pages for a book that exists because it's in the ABL, and the fact that the titles don't match really doesn't matter. Perhaps remove this assertion check? 🤷‍♂️